bibtex/bib/quantum.bib

```diff
@@ -810,7 +1047,7 @@ @InProceedings{Liu15
   Url                      = {http://arxiv.org/abs/1410.3918}
 }
 
-@InProceedings{Liu14,
+@InProceedings{Liu14a,
```

Comment:
My JabRef doesn't add "a" to the first article which results in the same citation key